src/daft-distributed/src/pipeline_node/translate.rs

```diff
@@ -1,12 +1,134 @@
+use std::sync::Arc;
+
 use common_daft_config::DaftExecutionConfig;
 use common_error::DaftResult;
-use daft_local_plan::LocalPhysicalPlanRef;
-use daft_logical_plan::LogicalPlanRef;
+use common_scan_info::{PhysicalScanInfo, ScanState};
+use daft_local_plan::{translate, LocalPhysicalPlanRef};
+use daft_logical_plan::{
+    ops::{
+        ActorPoolProject, Distinct, Explode, Filter, Join, Limit, Project, Repartition, Sample,
+        Sink, Sort, Source, Unpivot,
+    },
+    optimization::OptimizerBuilder,
+    LogicalPlan, LogicalPlanRef, SourceInfo,
+};
 
-#[allow(dead_code)]
 pub(crate) fn translate_pipeline_plan_to_local_physical_plans(
-    _logical_plan: LogicalPlanRef,
-    _execution_config: &DaftExecutionConfig,
+    logical_plan: LogicalPlanRef,
+    execution_config: &DaftExecutionConfig,
 ) -> DaftResult<Vec<LocalPhysicalPlanRef>> {
```

Comment:
separate this into a LocalPhysicalPlanRef and the inputs that can be patched somewhere.